x-pack/plugins/apm/public/components/shared/charts/transaction_charts/helper.tsx

```diff
@@ -24,12 +23,19 @@ export function getResponseTimeTooltipFormatter(formatter: TimeFormatter) {
   };
 }
 
-export function getMaxY(timeSeries: TimeSeries[]) {
-  const coordinates = flatten(
-    timeSeries.map((serie: TimeSeries) => serie.data as Coordinate[])
-  );
+export function getMaxY(
+  timeSeries?: Array<TimeSeries<Coordinate>> | TimeSeries<Coordinate>
```

Comment:
Again, I don't think this needs to handle both cases (unless it makes things worse somewhere else)